stdlib/http/src/main/java/org/ballerinalang/net/http/websocket/client/InitEndpoint.java

```diff
@@ -137,6 +133,63 @@ private static void populateClientConnectorConfig(MapValue<String, Object> clien
         return customHeaders;
     }
 
+    /**
+     * Populate the retry config.
+     *
+     * @param retryConfig - the retry config.
+     * @param retryConnectorConfig - the retry connector config.
+     */
+    private static void populateRetryConnectorConfig(MapValue<String, Object> retryConfig,
+                                                     RetryContext retryConnectorConfig) {
+        retryConnectorConfig.setInterval(getIntValue(retryConfig, "intervalInMillis", 1000));
+        retryConnectorConfig.setBackOfFactor(getDoubleValue(retryConfig));
+        retryConnectorConfig.setMaxInterval(getIntValue(retryConfig, "maxWaitIntervalInMillis", 30000));
+        retryConnectorConfig.setMaxAttempts(getIntValue(retryConfig, "maxCount", 0));
```

Comment:
Shall we use constants for these fields?

Reply:
As per the comment https://github.com/ballerina-platform/ballerina-lang/pull/19723#discussion_r372186211, 
I have directly used these constant because these are used only as a place.

Reply:
That is for statements. This comment is for variables: https://github.com/ballerina-platform/ballerina-lang/pull/19723#discussion_r372180320 
I think you have misunderstood where to use constants.